fluent-react/src/useTranslate.ts

```diff
@@ -0,0 +1,28 @@
+import { useContext } from "react";
+import { FluentContext } from "./context";
+import { ReactLocalization } from "./localization";
+
+// Implementation of null object pattern for ReactLocalization
+const fallbackL10n: ReactLocalization = {
+  getString: id => id,
+  getBundle: () => null,
+  bundles: [],
+  parseMarkup: null,
+  reportError: () => undefined,
+};
+
+/*
+* The `useTranslate` hook returns the FluentContext
+*/
+type useTranslate = () => { l10n: ReactLocalization }
+export const useTranslate: useTranslate = () => {
```

Comment:
Would you mind renaming this to `useLocalization`, to keep it in sync with the `withLocalization` HOC?

Reply:
Okay. Makes sense to me.